test/integration/node/lifecycle_test.go

```diff
@@ -113,7 +114,8 @@ func TestEvictionForNoExecuteTaintAddedByUser(t *testing.T) {
 					},
 				},
 			}
-
+			// TODO: this will be removed in 1.37
```

Comment:
Drop this comment, this will automatically fail compilation once you drop the feature gate, where I want you to move that comment.